model/value_float_test.go

```diff
@@ -291,3 +296,16 @@ func TestMatrixJSON(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkJSONMarshallingSamplePairMatrix(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		s, err := json.Marshal(samplePairMatrixValue)
+		if err != nil {
+			b.Fatal("error marshalling")
+		}
+
+		if string(s) != samplePairMatrixPlain {
+			b.Fatalf("encoding error: expected %s, got %s", samplePairMatrixPlain, s)
+		}
```

Comment:
We don't need this check in the benchmark; the correctness check belongs to unit tests, benchmark should only check performance. Can you remove this and re-run the benchmark and see if the numbers change? (either way, we should remove this)